test/elife-loader.spec.js

```diff
@@ -0,0 +1,92 @@
+const chai = require('chai');
+const sinon = require('sinon');
+
+const expect = chai.expect;
+const spy = sinon.spy;
+
+// load in component(s) to be tested
+let loader = require('../assets/js/elife-loader');
```

Comment:
Is this needed? Delete if not. The load looks to be handled by https://github.com/elifesciences/pattern-library/pull/928/files#diff-4b3d23034489717c53e6b44e65e7fdc0R23.

Reply:
Completed